packages/sdk/src/sdk.ts

```diff
@@ -62,7 +62,7 @@ export class SDK {
     fontConnector: FontConnectorController;
     animation: AnimationController;
     document: DocumentController;
-    configuration: ConfigurationController;
+    configuration: LocalConfigurationDecorator;
```

Comment:
Consider hiding this implementation detail.

```suggestion
    configuration: ConfigurationController;
```